03week/towersOfHanoi.js

```diff
@@ -19,24 +29,66 @@ function printStacks() {
   console.log("c: " + stacks.c);
 }
 
-function movePiece() {
-  // Your code here
-
+function movePiece(startStack, endStack) {
+  if (startStack === 'a' && endStack === 'b') {
```

Comment:
for every stack and letter, you're checking if the stack is a,b, or c, twice, that's a lot of repetition that you can condense.